src/libs/algolia/fetchAlgolia/helpers/getCurrentVenuesIndex.test.ts

```diff
@@ -0,0 +1,22 @@
+import { LocationType } from 'features/search/enums'
+import { getCurrentVenuesIndex } from 'libs/algolia/fetchAlgolia/helpers/getCurrentVenuesIndex'
+
+describe('getCurrentVenuesIndex', () => {
+  it('should return algoliaVenuesIndexPlaylistSearchNewest when locationType is EVERYWHERE', () => {
+    const params = LocationType.EVERYWHERE
+    const result = getCurrentVenuesIndex(params)
+    expect(result).toEqual('algoliaVenuesIndexPlaylistSearchNewest')
+  })
+
+  it('should return algoliaVenuesIndexPlaylistSearch when locationType is not EVERYWHERE', () => {
```

Comment:
Je pense que j'aurai été plus loin pour que ça soit bien visuel et que j'aurai testé chaque type de localisation.

Reply:
Tu peux faire un petit `it.each` pour ne pas dupliquer le test :)